dashboard/test/controllers/levels_controller_test.rb

```diff
@@ -72,6 +72,55 @@ def create_level_with_rubric
     )
   end
 
+  test "should get filters" do
```

Comment:
I don't know if this test is the right way to test this or if I should be hard coding a list of values.

Reply:
A good rule of thumb is that tests shouldn't recreate the same logic as the code being tested, since if there's a bug the test wouldn't catch it. So in this case I'd be in favor of hardcoded values.

Reply:
Updated

Reply:
This is a difficult feature to test 🙂  your updated tests are better in one way, for the reason Winter mentioned. but now they are brittle, because they'll need to be updated any time a new level type or script is added. I think what you have here is probably fine, since I can't think of a way to make them less brittle without making them worse in some other way.